src/kits/tracker/ContainerWindow.cpp

```diff
@@ -1010,13 +968,41 @@ BContainerWindow::Init(const BMessage* message)
 	SetFlags(Flags() & ~B_NO_WORKSPACE_ACTIVATION);
 }
 
+void
+BContainerWindow::InitLayouts()
+{
+	BLayoutItem* item = 
+		fPoseViewHolder->GroupLayout()->AddView(0, fPoseView->TitleView());
+	BSize minSize = item->MinSize();
+	BSize maxSize = item->MaxSize();
+	item->SetExplicitMinSize(BSize(minSize.Width(), kTitleViewHeight));
+	item->SetExplicitMaxSize(BSize(maxSize.Width(), kTitleViewHeight));
+		
+	item = fCountContainer->GroupLayout()->AddView(fPoseView->CountView());
+	minSize = item->MinSize();
+	maxSize = item->MaxSize();
+	item->SetExplicitMinSize(BSize(kCountViewWidth, minSize.Height()));
+	item->SetExplicitMaxSize(BSize(kCountViewWidth, maxSize.Height()));
```

Comment:
Why is something like kCountViewWidth, and even worse, kTitleViewHeight needed? Shouldn't the layout take care of that automatically according to the font size specified?